### DISCUSSION TITLE
installation guide for apache2 vichan

### DISCUSSION THREAD

**Discussion by systemd1337** [vichan-devel/vichan#512] [Show and tell]
https://github.com/vichan-devel/vichan/wiki/installation-guide-for-vichan-ubuntu-20.04

updated feel free to update the rest. i prefer apache2 over nginx

**Comment by systemd1337** [comment]
aaannd updated again :D

**Comment by ctrlcctrlv** [comment]
you probably want to tell people to use mariadb not mysql.
